imgui/src/extension_imgui.cpp

```diff
@@ -2700,6 +2700,79 @@ static int imgui_PopStyleColor(lua_State* L)
     return 0;
 }
 
+/** PushStyleVar
+ * @name push_style_var
+ * @tparam number style_var enum for the style to be pushed
+ * @param value number or vmath.vector3 specifying the value of the style var
+ */
+static int imgui_PushStyleVar(lua_State *L)
+{
+    DM_LUA_STACK_CHECK(L, 0);
+    uint32_t style_var = luaL_checkint(L, 1);
+    switch(style_var)
+    {
+        case ImGuiStyleVar_Alpha:
+        case ImGuiStyleVar_DisabledAlpha:
+        case ImGuiStyleVar_WindowRounding:
+        case ImGuiStyleVar_WindowBorderSize:
+        case ImGuiStyleVar_ChildRounding:
+        case ImGuiStyleVar_ChildBorderSize:
+        case ImGuiStyleVar_PopupRounding:
+        case ImGuiStyleVar_PopupBorderSize:
+        case ImGuiStyleVar_FrameRounding:
+        case ImGuiStyleVar_FrameBorderSize:
+        case ImGuiStyleVar_IndentSpacing:
+        case ImGuiStyleVar_ScrollbarSize:
+        case ImGuiStyleVar_ScrollbarRounding:
+        case ImGuiStyleVar_GrabMinSize:
+        case ImGuiStyleVar_GrabRounding:
+        case ImGuiStyleVar_TabRounding:
+        case ImGuiStyleVar_TabBorderSize:
+        case ImGuiStyleVar_TabBarBorderSize:
+        case ImGuiStyleVar_TableAngledHeadersAngle:
+        case ImGuiStyleVar_SeparatorTextBorderSize:
+        {
+            float val_float = luaL_checknumber(L, 2);
+            ImGui::PushStyleVar(style_var, val_float);
+            break;
+        }
+        case ImGuiStyleVar_WindowPadding:
+        case ImGuiStyleVar_WindowMinSize:
+        case ImGuiStyleVar_WindowTitleAlign:
+        case ImGuiStyleVar_FramePadding:
+        case ImGuiStyleVar_ItemSpacing:
+        case ImGuiStyleVar_ItemInnerSpacing:
+        case ImGuiStyleVar_CellPadding:
+        case ImGuiStyleVar_TableAngledHeadersTextAlign:
+        case ImGuiStyleVar_ButtonTextAlign:
+        case ImGuiStyleVar_SelectableTextAlign:
+        case ImGuiStyleVar_SeparatorTextAlign:
+        case ImGuiStyleVar_SeparatorTextPadding:
+        {
+            dmVMath::Vector3* val_vec3 = dmScript::CheckVector3(L, 2);
+            ImVec2 val_vec2(val_vec3->getX(), val_vec3->getY());
+            ImGui::PushStyleVar(style_var, val_vec2);
+            break;
+        }
+    }
+    return 0;
+}
+
+/** PopStyleVar
+ * @name pop_style_var
+ */
```

Comment:
Missing `@number count Number of values to pop`

Reply:
Done.